docs/4.2/features/enhanced_session_recording.md

```diff
@@ -243,4 +250,60 @@ ssh_service:
 ```
 
 ## 5. Inspect Logs
-The resulting enhanced session recording will be shown in the logs. 
\ No newline at end of file
+The resulting enhanced session recording will be shown in [Teleports Audit Log](https://gravitational.com/teleport/docs/architecture/teleport_auth/#audit-log).
+
+
+
+```bash
+$ teleport-auth ~:  tree /var/lib/teleport/log
+/var/lib/teleport/log
+├── 1048a649-8f3f-4431-9529-0c53339b65a5
+│   ├── 2020-01-13.00:00:00.log
+│   └── sessions
+│       └── default
+│           ├── fad07202-35bb-11ea-83aa-125400432324-0.chunks.gz
+│           ├── fad07202-35bb-11ea-83aa-125400432324-0.events.gz
+│           ├── fad07202-35bb-11ea-83aa-125400432324-0.session.command-events.gz
+│           ├── fad07202-35bb-11ea-83aa-125400432324-0.session.network-events.gz
+│           └── fad07202-35bb-11ea-83aa-125400432324.index
+├── events.log -> /var/lib/teleport/log/1048a649-8f3f-4431-9529-0c53339b65a5/2020-01-13.00:00:00.log
+├── playbacks
+│   └── sessions
+│       └── default
+└── upload
+    └── sessions
+        └── default
+```
+
+To quickly check the status of the audit log. You can simply tail the logs with 
```

Comment:
`audit log. You can` -> `audit log, you can` (comma rather than full stop, sentence case)